lib/fathead/sass/output.txt

```diff
@@ -0,0 +1,233 @@
+Nested Properties	A										CSS has quite a few properties that are in “namespaces;” for instance, font-family, font-size, and font-weight are all in the font namespace. In CSS, if you want to set a bunch of properties in the same namespace, you have to type it out each time. Sass provides a shortcut for this: just write the namespace once, then nest each of the sub-properties within it. For example:<pre class="code ruby"><code class="ruby">.funky {\n  font: {\n    family: fantasy;\n    size: 30em;\n    weight: bold;\n  }\n}</code></pre>	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_properties
+Placeholder Selectors %foo	A										Sass supports a special type of selector called a “placeholder selector”. These look like class and id selectors, except the # or . is replaced by %. They’re meant to be used with the @extend directive; for more information see @extend-Only Selectors.	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_
+Comments /* */ and //	A										Sass supports standard multiline CSS comments with /* */, as well as single-line comments with //. The multiline comments are preserved in the CSS output where possible, while the single-line comments are removed. For example:<pre class="code ruby"><code class="ruby">/* This comment is\n * several lines long.\n * since it uses the CSS comment syntax,\n * it will appear in the CSS output. */\nbody { color: black; }\n\n// These comments are only one line long each.\n// They won't appear in the CSS output,\n// since they use the single-line comment syntax.\na { color: green; }</code></pre>	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments
+SassScript	A										In addition to the plain CSS property syntax, Sass supports a small set of extensions called SassScript. SassScript allows properties to use variables, arithmetic, and extra functions. SassScript can be used in any property value.	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#sassscript
+Interactive Shell	A										You can easily experiment with SassScript using the interactive shell. To launch the shell run the sass command-line with the -i option. At the prompt, enter any legal SassScript expression to have it evaluated and the result printed out for you:<pre class="code ruby"><code class="ruby">$ sass -i\n&gt;&gt; "Hello, Sassy World!"\n"Hello, Sassy World!"\n&gt;&gt; 1px + 1px + 1px\n3px\n&gt;&gt; #777 + #777\n#eeeeee\n&gt;&gt; #777 + #888\nwhite</code></pre>	http://sass-lang.com/documentation/file.SASS_REFERENCE.html#interactive_shell
```

Comment:
Can you see why this contains `&gt;&gt;` instead of `>>` -- we don't need to escape HTML entities 👍